domains/bion.is-cool.dev.json

```diff
@@ -0,0 +1,16 @@
+{
+    "description": "Event Portfolio Website",
+
+    "domain": "is-cool.dev",
+    "subdomain": "bion",
+
+    "owner": {
+        "email": "raannakasturi@gmail.com"
+    },
+
+    "record": {
+        "NS": ["ns1.infinityfree.com", "ns2.infinityfree.com"]
```

Comment:
We do not longer accept new NS record as per our readme.md.

Reply:
A: 185.27.134.3